11-React/.docker/docker-compose.yaml

```diff
@@ -0,0 +1,11 @@
+services:
+  frontend:
+    build:
+      context: .
+      target: dev-envs
+    container_name: frontend
+    ports:
+      - "80:3000"
+      - "3000:3000"
```

Comment:
Why did u open port 3000?
only one port is required for client applications

Reply:
port 3000 is opened to allow local access to the React development server and port 80 is opened for access to the service over the internet.
 i fix it